yarn.lock

```diff
@@ -11059,6 +11064,35 @@ webpack@4.35.2:
     watchpack "^1.5.0"
     webpack-sources "^1.3.0"
 
+webpack@4.35.3:
```

Comment:
Looks like it will install two versions of `webpack` package now (second record is above). Shall the version be updated in some other package as well?

Reply:
Good catch!